network/topics/params/peer_score.go

```diff
@@ -57,6 +57,11 @@ func PeerScoreParams(oneEpoch, msgIDCacheTTL time.Duration, ipWhilelist ...*net.
 	targetVal = targetVal - behaviourPenaltyThreshold
 	behaviourPenaltyWeight := gossipThreshold / (targetVal * targetVal)
 
+	localIPColocationFactorWeight := ipColocationFactorWeight
```

Comment:
As I understand, it's assumed to be local when `disableCoLocation` is true? If so, the name might be confusing as it's not always local